### PR TITLE
Sphinx doc chapter 28

### PR DESCRIPTION
**Pull request by maximedenes**

### CODE REVIEW

File: doc/sphinx/addendum/parallel-proof-processing.rst [hunk 1]
```diff
@@ -1,221 +1,229 @@
...
+Proof block detection is performed by looking at the syntax of the
+proof script (i.e. also looking at indentation). |Coq| comes with four
+kind of proof blocks, and an ML API to add new ones.
+
+:curly: blocks are delimited by { and }, see Chapter :ref:`proofhandling`
+:par: blocks are atomic, i.e. just one tactic introduced by the `par:`
+  goal selector
+:indent: blocks end with a tactic indented less than the previous one
+:bullet: blocks are delimited by two equal bullet signs at the same
+  indentation level
```
Comment (Zimmi48) [inline doc/sphinx/addendum/parallel-proof-processing.rst]: Interesting, I didn't know there were already parts of Coq which were indentation-dependent.

File: doc/sphinx/addendum/parallel-proof-processing.rst [hunk 2]
```diff
@@ -1,221 +1,229 @@
...
-\texttt{.vio} files are complete (they contain all proof terms and
-universe constraints), the satisfiability of all universe constraints has
-not been checked globally (they are checked to be consistent for every
-single proof). Constraints will be checked when these \texttt{.vo} files
-are (recursively) loaded with \texttt{Require}.
+compiled file (like the `.vo` one) that can be loaded by ``Require`` from the
+generation and checking of the proof objects. The ``-quick`` flag can be
+passed to `coqc` or `coqtop` to produce, quickly, `.vio` files.
+Alternatively, when using a Makefile produced by `coq_makefile`,
+the *quick* target can be used to compile all files using the ``-quick`` flag.
```
Comment (Zimmi48) [inline doc/sphinx/addendum/parallel-proof-processing.rst]: Both quick should be rendered the same here: no italic.

File: doc/sphinx/addendum/parallel-proof-processing.rst [hunk 3]
```diff
@@ -1,221 +1,229 @@
...
+checked.
+
+When working interactively, one can fully check all the `.v` files by
+running `coqc` as usual.
+
+Alternatively one can turn each `.vio` into the corresponding `.vo`. All
+.vio files can be processed in parallel, hence this alternative might
+be faster. The command ``coqtop -schedule-vio2vo 2 a b c`` can be used to
+obtain a good scheduling for two workers to produce `a.vo`, `b.vo`, and
+`c.vo`. When using a Makefile produced by `coq_makefile`, the *vio2vo* target
```
Comment (Zimmi48) [inline doc/sphinx/addendum/parallel-proof-processing.rst]: Same thing for the make target here.

File: doc/sphinx/addendum/parallel-proof-processing.rst [hunk 4]
```diff
@@ -1,221 +1,229 @@
...
-%%% End: 
+proof tasks stored in `.vio` files without producing the `.vo` files. This
+is possibly faster because all the proof tasks are independent, hence
+one can further partition the job to be done between workers. The
+``coqtop -schedule-vio-checking 6 a b c`` command can be used to obtain a
+good scheduling for 6 workers to check all the proof tasks of `a.vio`,
+`b.vio`, and `c.vio`. Auxiliary files are used to predict how long a proof
+task will take, assuming it will take the same amount of time it took
+last time. When using a Makefile produced by coq_makefile, the
+*checkproofs* target can be used to check all `.vio` files. Variable `J`
```
Comment (Zimmi48) [inline doc/sphinx/addendum/parallel-proof-processing.rst]: Same. I don't know why this choice to put make targets in italic was made.

**Comment by maximedenes** [comment]
Comments addressed, rebased on other Sphinx PRs.

**Comment by Zimmi48** [comment]
There are many end-of-line whitespaces in the new Sphinx file, as reported by Travis. Could you fix them?

**Comment by maximedenes** [comment]
> There are many end-of-line whitespaces in the new Sphinx file, as reported by Travis. Could you fix them?

Should be good now, thanks!

**Comment by Zimmi48** [comment]
> Should be good now, thanks!

Travis seems to think otherwise...

**Comment by maximedenes** [comment]
I thought I had `rebase --whitespace=fix`, but let's try again.

**Comment by maximedenes** [comment]
But the linter reports problem in other chapters, right?

**Comment by maximedenes** [comment]
Why can't I see if there are conflicts or not? What is this "Required statuses must pass before merging"?

**Comment by Zimmi48** [comment]
There are no conflicts. Otherwise, it would show them. I don't know anything about this "Required statuses"

**Comment by Zimmi48** [comment]
But if you rebase now on master, it should only keep the present chapter, given that I merged the previous ones.

**Comment by SkySkimmer** [comment]
It's saying that CI failed and the code owner review is pending so you can't merge by clicking the github button.

**Comment by maximedenes** [comment]
> But if you rebase now on master, it should only keep the present chapter, given that I merged the previous ones.

Done.

**Comment by maximedenes** [comment]
@Zimmi48 can you merge?

**Pull request merged**